gpkit/constraints/relax.py

```diff
@@ -0,0 +1,140 @@
+"""Models for assessing primal feasibility"""
+from .model import Model
+from .set import ConstraintSet
+from ..nomials import Variable, VectorVariable, parse_subs, NomialArray
+
+
+class RelaxAll(Model):
```

Comment:
maybe make it a method of `Model`, and implement it by forming a `RelaxConstraints` and changing the cost? Or cut entirely?